interface/patient_file/encounter/Services/ViewHelper.php

```diff
@@ -90,7 +90,8 @@ static function createEncounterMenu(array $elements)
         $submenuListItem = '<li><a href="{href}" class="dropdown-toggle" aria-haspopup="true" aria-expanded="false">{linkText}</a>';
 
         // Standard menu item dropdown support
-        $menuListItemWithDropdown = '<li class="dropdown"><a href="{href}" class="dropdown-toggle" aria-haspopup="true" aria-expanded="false"><i class="fa fa-fw fa-chevron-right"></i>&nbsp;{linkText}</a>{submenuList}</li>';
+        $iconChevronDirection = $_SESSION['language_direction'] == 'rtl' ? 'fa-chevron-left' : 'fa-chevron-right';
+        $menuListItemWithDropdown = '<li class="dropdown"><a href="{href}" class="dropdown-toggle" aria-haspopup="true" aria-expanded="false"><i class="fa fa-fw ' . $iconChevronDirection .'"></i>&nbsp;{linkText}</a>{submenuList}</li>';
```

Comment:
Please avoid breaking strings like this. Instead, follow the convention used in the rest of this function and use a `str_replace` function. I'll fix this in my branch.